### DISCUSSION TITLE
ase calculator

### DISCUSSION THREAD

**Discussion by chen-jialan** [mir-group/allegro#23] [Q&A] [answered]
Hi,
I want to use this model for calculator some system with ase, if or not the code for interface of ase?

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @chen-jialan ,

See https://github.com/mir-group/nequip/discussions/115 for an example of using `nequip.ase.NequIPCalculator`, which works with Allegro in the exact same way.

**Reply by miroi** [reply]
Hello, I am looking for some ase-allegro example, is there some please ? See https://github.com/mir-group/nequip/issues/470 and https://github.com/mir-group/nequip/issues/471 .
